docs/release-notes/8.8.asciidoc

```diff
@@ -0,0 +1,97 @@
+* Adds an expandable section to the alert details flyout with various additional features and information. NOTE: to access this functionality you currently need to add a line to the `kibana.json` file. {pull}152935[#152935], {pull}152767[#152767], {pull}152303[#152303], {pull}152150[#152150], {pull}152047[#152047], {pull}150240[#150240], {pull}153807[#153807], {pull}153187[#153187], {pull}153706[#153706], {pull}153074[#153074], {pull}154196[#154196], {pull}153903[#153903], {pull}154114[#154114]).
+* Adds a "visualizations" section to the Alert details flyout ({pull}154786[#154786]).
+* Adds Analyzer to the expanded alert details flyout ({pull}153709[#153709]).
+* Adds Session View to the expanded alert details flyout ({pull}154597[#154597]).
+
+
+LENS update: visualizations throughout security have been updated to Lens visualizations, and can be opened in Lens
+
+[discrete]
+[[enhancements-8.8.0]]
+==== Enhancements
+
+* Updates the Notable anomalies table, and renames it to Anomalies ({pull}155687[#155687]).
+* Updates the Data Quality dashboard to include a new tree map, and storage size metrics for each index ({pull}155581[#155581]).
+* Displays additional anomaly jobs on the Entity Analytics dashboard ({pull}155520[#155520]).
+* Adds cloud infrastructure-related fields to the alert details flyout's highlighted fields ({pull}155247[#155247]).
+* Allows you to specify how to handle alert suppression for alerts with missing fields ({pull}155055[#155055]).
+* Updates the rule settings Actions interface ({pull}154526[#154526]).
+* Adds a warning message to tell you when the maximum number of alerts for a single rule execution is exceeded ({pull}154112[#154112]).
+* Updates how browser field descriptions are provided to {kib} ({pull}153498[#153498]).
+* Makes alert links on the Entity Analytics page navigate to the Alerts page instead of opening in Timeline ({pull}153372[#153372]).
+* Enables multi-level grouping for alerts on the Alerts page, based on various fields ({pull}152862[#152862]).
+* Makes it possible to link directly to a version of the Alerts page with filters active, and makes widgets on the Detection and Response dashboard do so ({pull}152714[#152714]).
+* Updates the visualizations throughout {elastic-sec} to Lens visualizations ({pull}150531[#150531]).
+* Adds a "Share alert" button to the alert details flyout, and makes alert flyout URLs more sharable ({pull}148800[#148800]).
+* Adds a note to the Rules page when a maintenance window is running ({pull}155386[#155386]).
+* Adds a global search bar to the Detections and Response and Entity Analytics dashboards ({pull}156832[#156832]).
+
+
+[discrete]
+[[bug-fixes-8.8.0]]
+==== Bug fixes
+
+* Fixes a bug that interfered with the default time range when you opened an alert in Timeline ({pull}156884[#156884]).
+* Updates where the technical preview tags appear for host risk score features ({pull}156659[#156659], {pull}156514[#156514]).
```

Comment:
This doesn't seem like a bug fix; maybe an enhancement? @jmikell821 are the risk scores still tech preview or are they GA this release?